packages/dart-board/package.json

```diff
@@ -53,7 +53,7 @@
   "dependencies": {
     "@ugrc/utilities": "^2.2.1",
     "clsx": "^1.2.1",
-    "reactstrap": "^9.1.4"
+    "reactstrap": "^9.1.5"
```

Comment:
**thought**: i always wonder if we should be loose on these widget dependencies. Like `> 2.0.0` etc.

Reply:
I didn't know that you could specify npm deps like that. Hmmm

Reply:
don't we do it with react that way?